app/views/content/life-as-a-teacher/explore-subjects/modern-languages.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Become a modern languages teacher
+title_paragraph: |-
+  <p>
+  By teaching pupils modern languages, you can help them gain a greater appreciation and understanding of the world. As they learn French, Spanish or German, your pupils will get a unique insight into the countries that speak these languages.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_languagesother$ or scholarships of $scholarships_languagesfrenchgermanspanish$ are available for eligible trainee modern languages teachers.</p>
```

Comment:
I think this needs wording more carefully - scholarships are only available for French, German and Spanish - bursaries are available for all modern languages